src/main/java/org/wildfly/swarm/microprofile/fault/tolerance/hystrix/HystrixCommandInterceptor.java

```diff
@@ -47,33 +47,30 @@
 @Priority(Interceptor.Priority.LIBRARY_AFTER + 1)
 public class HystrixCommandInterceptor {
 
+    private final Map<Method, Setter> methodToSetterMap = new ConcurrentHashMap<>();
 
     @AroundInvoke
     public Object interceptCommand(InvocationContext ic) throws Exception {
 
         Method method = ic.getMethod();
+
         ExecutionContextWithInvocationContext ec = new ExecutionContextWithInvocationContext(ic);
         Asynchronous async = getAnnotation(method, Asynchronous.class);
-        Timeout Timeout = getAnnotation(method, Timeout.class);
-        CircuitBreaker circuitBreaker = getAnnotation(method, CircuitBreaker.class);
-        Retry retry = getAnnotation(method, Retry.class);
-        Fallback fallback = getAnnotation(method, Fallback.class);
-        HystrixCommandProperties.Setter setter = HystrixCommandProperties.Setter();
-        Supplier fallbackToRun = null;
 
-        if (Timeout != null) {
-            long Timeoutmillis = Duration.of(Timeout.value(), Timeout.unit()).toMillis();
-            setter = setter.withExecutionTimeoutInMilliseconds((int) Timeoutmillis);
-        }
+        // TODO
+        // CircuitBreaker circuitBreaker = getAnnotation(method, CircuitBreaker.class);
+        // Retry retry = getAnnotation(method, Retry.class);
+
+        Fallback fallback = getAnnotation(method, Fallback.class);
+        Supplier<Object> fallbackToRun = null;
 
         if (fallback != null) {
-            FallbackHandler fbh = fallback.value().newInstance();
+            FallbackHandler<?> fbh = fallback.value().newInstance();
```

Comment:
I think this part will be completely rewritten. See also https://github.com/eclipse/microprofile-fault-tolerance/issues/4#issuecomment-308676325